packages/webapp/src/components/Map/SelectionHandler/index.jsx

```diff
@@ -139,6 +139,11 @@ export default function PureSelectionHandler({ locations, history, sensorReading
     );
   };
 
+  const isSensor = (id) => {
+    let sensor = readingTypes.find((sensor) => sensor.location_id === id);
+    return sensor?.reading_types;
```

Comment:
`let` -> `const`
I think it is better to return boolean: `return !!sensor?.reading_types`